virtualpdu/pdu/baytech_mrp27.py

```diff
@@ -0,0 +1,53 @@
+# Copyright 2016 Internap
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from pyasn1.type import univ
+
+from virtualpdu import core
+from virtualpdu.pdu import BasePDUOutletStates
+from virtualpdu.pdu import PDU
+from virtualpdu.pdu import PDUOutlet
+
+rPDU_outlet_control_outlet_command = \
```

Comment:
Thats most likely not the right name for baytech. Refer to the mib for the exact name.